jpa/subsystem/src/main/java/org/jboss/as/jpa/transaction/TransactionUtil.java

```diff
@@ -45,23 +52,51 @@
 
     private static volatile TransactionSynchronizationRegistry transactionSynchronizationRegistry;
     private static volatile TransactionManager transactionManager;
+    private static volatile TransactionListenerRegistry transactionListenerRegistry;
```

Comment:
We used to have a (static) way for standalone code to lookup a service, but I think we eliminated those calls.  We have the (singleton) JPAService that is responsible for the state of these statics.  However, we need access to these statics from code that does not have a way to lookup the JPAService (e.g. in a container managed entity manager, which is called many times from applications).  I'm not worried about the thousands of (EntityManager) calls that need access to the JPAService as much as we need a static way to look it up (I thought we deleted the code that allowed that a few years ago).